### PR TITLE
Multi-cluster support

### PR DESCRIPTION
**Pull request by srkirkland**
Handles structure of multi-cluster support, including all user activity and some example admin activity.

All API endpoints client/server have been updated to take cluster as url param.  Components have been updated to check this param and controllers expose it.  LINQ queries now use custom `InCluster()` filter for most queries.

A few admin functions still need to be updated (see TODO in adminController.cs).  Additionally admin permissions checked by auth policy need to be updated to handle cluster as well.

### CODE REVIEW

File: Hippo.Core/Models/AccountDetail.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{
+    public class AccountDetail
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Status { get; set; }
+        public bool CanSponsor { get; set; }
+        public string Owner { get; set; }
+        public string Cluster { get; set; }
+        public string Sponsor { get; set; }
```
Comment (jSylvestre) [inline Hippo.Core/Models/AccountDetail.cs:13]: Need IsAdmin

File: Hippo.Web/ClientApp/src/components/Account/RequestForm.tsx [hunk 2]
```diff
@@ -66,7 +72,7 @@ export const RequestForm = () => {
           <span className="status-color">{context.user.detail.firstName}</span>
         </h3>
         <p>
-          You don't seem to have an account on Farm yet. If you'd like access,
+          You don't seem to have an account on the {cluster} cluster yet. If you'd like access,
```
Comment (jSylvestre) [inline Hippo.Web/ClientApp/src/components/Account/RequestForm.tsx:75]: Worthwhile getting the cluster description here?

File: Hippo.Web/ClientApp/src/components/Admin/AdminUsers.tsx [hunk 3]
```diff
@@ -117,17 +119,17 @@ export const AdminUsers = () => {
...
+                  <td>{account.owner?.email}</td>
                   <td>
                     <button
                       disabled={notification.pending}
-                      onClick={() => handleRemove(user)}
+                      onClick={() => handleRemove(account)}
                       className="btn btn-primary"
                     >
-                      {adminRemoving === user.id ? "Removing..." : "Remove"}
+                      {adminRemoving === account.id ? "Removing..." : "Remove"}
```
Comment (jSylvestre) [inline Hippo.Web/ClientApp/src/components/Admin/AdminUsers.tsx:132]: I think this still needs to be the user id? account.owner.id

**Review by jSylvestre — APPROVED** [review]

File: Hippo.Web/ClientApp/src/components/Account/RequestForm.tsx (hunk 2 above)
Comment (srkirkland) [reply Hippo.Web/ClientApp/src/components/Account/RequestForm.tsx:75]: i didn't think it was worth it.  but if we put it in the context then maybe..

File: Hippo.Web/ClientApp/src/components/Admin/AdminUsers.tsx (hunk 3 above)
Comment (srkirkland) [reply Hippo.Web/ClientApp/src/components/Admin/AdminUsers.tsx:132]: not sure but we probably want to move to account ids and away from owner useage

Comment (jSylvestre) [reply Hippo.Web/ClientApp/src/components/Admin/AdminUsers.tsx:132]: @srkirkland This was my mistake. This needs to be account.id. But I had to change the post to use user.id.
Fixed in my pr

**Pull request merged**